src/main/scala/com/campudus/tableaux/database/DatabaseQuery.scala

```diff
@@ -190,6 +190,11 @@ class RowStructure(connection: DatabaseConnection) {
     connection.singleQuery(s"INSERT INTO user_table_$tableId DEFAULT VALUES RETURNING id", Json.arr())
   } map { insertNotNull(_).get[JsonArray](0).get[IdType](0) }
 
+  def createFull(tableId: IdType, values: Seq[_]): Future[IdType] = {
+    val v = values.foldLeft(Seq[String]())((s, _) => s :+ "?").mkString(", ")
+    connection.singleQuery(s"INSERT INTO user_table_$tableId VALUES (DEFAULT, $v) RETURNING id", values.foldLeft(Json.arr())((json, v) => json.add(v)))

```

Comment:
`: _*` expands a `Seq`, so it can be used in the varargs that `Json.arr()` expects.

If you have `def someMethod(someStrings: String*): Unit = println(someStrings.mkString("\n"))` for example, you can call it with a `List[String]` or `Seq[String]` etc by calling `someMethod(myList: _*)`.